src/Workspaces/Core/Portable/Utilities/ObjectPools/PooledObject.cs

```diff
@@ -36,50 +37,37 @@ public void Dispose()
         #region factory
         public static PooledObject<StringBuilder> Create(ObjectPool<StringBuilder> pool)
         {
-            return new PooledObject<StringBuilder>(
-                pool,
-                p => Allocator(p),
-                (p, sb) => Releaser(p, sb));
+            return new PooledObject<StringBuilder>(pool, Allocator, Releaser);
```

Comment:
❗️ This introduces two allocations where previously there were zero. See #18924.

Reply:
ah. that is why it was written that way. alright, I will move back to old style.